report.py

```diff
@@ -1,12 +1,16 @@
-import jinja2
-from smtplib import SMTP
-from email.mime.text import MIMEText
+import json
 from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from smtplib import SMTP
+from urllib.parse import quote
+
 from functions import generate_html_file, get_bugs_dict, \
 	get_bugs_set, get_jenkins_job_info, get_jenkins_jobs, \
 	get_jira_dict, get_jira_set, get_osp_version, \
 	get_other_blockers, percent
 
+import jinja2
```

Comment:
Move between lines 1 and 3

Reply:
I'm following pep8 styling guide for import statements and there is a plugin to flake8 called flake8-import-order [1] which checks this, you can add it to the requirements/test-requirements.txt 
From pep8 guide [2]
```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.
```
[1] https://pypi.org/project/flake8-import-order/
[2] https://www.python.org/dev/peps/pep-0008/#imports

Reply:
If you wish to incorporate this standard as part of the project (while I see the usefulness of PEP8 I find some of the "standards" silly - this one seems harmless enough I suppose) please incorporate it as part of the project CI 

Only standards that are part of the CI will be considered